tests/plugins/test_koji_promote.py

```diff
@@ -1570,6 +1572,33 @@ def test_koji_promote_no_parent_id(self, tmpdir, os_env, reactor_config_map):  #
         assert 'parent_build_id' not in image.keys()
         assert PLUGIN_KOJI_PARENT_KEY not in workflow.prebuild_results
 
+    def test_koji_promote_owner_submitter(self, tmpdir, monkeypatch):
+        session = MockedClientSession('')
+        session.getTaskInfo = lambda x: {'owner': 1234}
```

Comment:
You probably want to validate that this function was called, how about:
```python
flexmock(session).should_receive('getTaskInfo').with_args(1234).and_return({'owner': 456789})
```

*You probably want to use a different value for task ID and user ID.